### DISCUSSION TITLE
springcloud-mall/demo/page

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#705] [Announcements]
# springcloud-mall/demo/page

more 注释之前的内容被视为文章摘要。 页面信息 你可以在 Markdown 的 Frontmatter 中设置页面信息。 作者设置为 Ms.Hope。; 写作日期为 2020 年 1 月 1 日; 分类为 “使用指南”; 标签为 “页面配置” 和 “使用指南”; 页面内容 你可以自由在这里书写你的 Markdown。 你可以将图片和 Markdow...

http://localhost:8080/springcloud-mall/demo/page.html

**Comment by czshh0628** [comment]
测试评论
